azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/OSProfile.java

```diff
@@ -91,15 +90,52 @@ public static LinuxConfiguration create(final String disablePasswordAuthenticati
 
       @AutoValue
       public abstract static class WinRM {
+          public enum Protocol {
+
+              HTTP("http"),
+              HTTPS("https"),
+              UNRECOGNIZED("Unrecognized");
```

Comment:
I do not think it is appropriate for this case since there are no other options.
Even if there are it will be invalid for any jclouds user.